juno/assetlist.json

```diff
@@ -206,7 +206,28 @@
         "symbol": "ASVT",
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/asvt.png"
-        }
+        },
```

Comment:
checks failed. this closing brace is for the logo URIs of ASVT. There still needs to be a closing brace for ASVT before JOE begins

Reply:
yes, you need to have a brace to close the asvt logo_URIs, and then another closing brace for the asvt asset.